src/Components/ReviewForm.js

```diff
@@ -43,13 +43,15 @@ export default function ReviewForm({
 
   let edited = false;
 
+  const typeSingular = type === "comments" ? "comment" : "Review";
```

Comment:
nit: Is it intentional to capitalize 'Review' but not 'comment'?  If not, which casing do you prefer?